includes/agentic-commerce/class-wc-stripe-agentic-commerce-order-mapper.php

```
@@ -412,6 +413,32 @@ private function store_stripe_metadata( WC_Order $order, WC_Stripe_Agentic_Check
+	 * @param WC_Order                           $order   The WooCommerce order.
+	 * @param WC_Stripe_Agentic_Checkout_Session $session The checkout session wrapper.
+	 */
+	private function store_order_attribution( WC_Order $order, WC_Stripe_Agentic_Checkout_Session $session ): void {
+		$agent_source = $session->get_agent_source();
+		if ( null === $agent_source ) {
+			return;
+		}
+
+		$order->update_meta_data( '_wc_order_attribution_source_type', 'referral' );
+		$order->update_meta_data( '_wc_order_attribution_utm_source', $agent_source );
+	}
```

PHPMD (2.15.0)

[error] The method store_order_attribution is not named in camelCase. (undefined)

(CamelCaseMethodName)